games-roguelike/hengband/hengband-1.6.2-r1.ebuild

```diff
@@ -44,21 +44,17 @@ src_configure() {
 	local myconf
 	use l10n_ja || myconf="--disable-japanese"
 
-	egamesconf \
-		--with-setgid=${GAMES_GROUP} \
+	econf \
 		$(use_with X x) \
 		${myconf}
 }
 
 src_install() {
-	make DESTDIR="${D}" install || die "make install failed"
+	emake DESTDIR="${D}" install || die "make install failed"
 	if use l10n_ja ; then
 		dodoc readme.txt autopick.txt readme_eng.txt autopick_eng.txt
 	else
 		newdoc readme_eng.txt readme.txt
 		newdoc autopick_eng.txt autopick.txt
 	fi
-	prepgamesdirs
```

Comment:
It seems odd to be able to drop this but not have to do some fiddling with paths in the build system

Reply:
Yes i thought it was odd too! I think you're right that the answers probably lie herein

Reply:
hrm, the default should do `make DESTDIR=${D} install`, which should enough to install this.

There's something fishy going on with `--with-setgid` config parameter, as it seems this needs to be used in order for installation to do anything.

I'm going to step away from this ebuild for a bit and come back with fresh eyes, perhaps working on some other  ones will help illuminate for me what I'm missing here.